### PR TITLE
Syncman tests

### PR DESCRIPTION
**Pull request by KavishShah09**
Task linked: https://app.clickup.com/t/37rv46

### CODE REVIEW

**Comment by sharadregoti** [comment]
@KavishShah09 is this done?

**Comment by KavishShah09** [comment]
@sharadregoti this is done

File: gateway/utils/syncman/syncman_crud.go [hunk 1]
```diff
@@ -106,14 +106,15 @@ func (s *Manager) GetLogicalDatabaseName(ctx context.Context, project, dbAlias s
 	return collection.DBName, nil
 }
 
+type response struct {
```
Comment (YourTechBud) [inline gateway/utils/syncman/syncman_crud.go:109]: Move this to `types.go`. Rename to `preparedQueryResponse`

File: gateway/utils/syncman/syncman_crud.go [hunk 2]
```diff
@@ -459,15 +460,16 @@ func (s *Manager) GetDatabaseConfig(ctx context.Context, project, dbAlias string
 	return services, nil
 }
 
+type rulesResponse struct {
```
Comment (YourTechBud) [inline gateway/utils/syncman/syncman_crud.go:463]: move this to `syncman/types.go`. Rename to `dbRulesResponse`

File: gateway/utils/syncman/syncman_crud.go [hunk 3]
```diff
@@ -477,31 +479,32 @@ func (s *Manager) GetCollectionRules(ctx context.Context, project, dbAlias, col
...
 	for dbName, dbInfo := range databases {
 		for key, value := range dbInfo.Collections {
-			coll[fmt.Sprintf("%s-%s", dbName, key)] = &response{IsRealTimeEnabled: value.IsRealTimeEnabled, Rules: value.Rules}
+			coll[fmt.Sprintf("%s-%s", dbName, key)] = &rulesResponse{IsRealTimeEnabled: value.IsRealTimeEnabled, Rules: value.Rules}
 		}
 	}
 	return []interface{}{coll}, nil
 }
 
+type schemaResponse struct {
```
Comment (YourTechBud) [inline gateway/utils/syncman/syncman_crud.go:501]: Move to `syncman/types.go`. Rename to `dbSchemaResponse`

File: gateway/utils/syncman/types.go [hunk 4]
```diff
@@ -0,0 +1,137 @@
+package syncman
```
Comment (YourTechBud) [inline gateway/utils/syncman/types.go:1]: Rename this file to `types_test`

**Pull request merged**